pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -480,31 +481,60 @@ class Build {
         }
         def sdkUrl = "${env.BUILD_URL}/artifact/workspace/target/${filter}/*zip*/target.zip"
         context.echo "sdkUrl is ${sdkUrl}"
-        def targets = ['sanity.jck', 'extended.jck', 'special.jck']
-        def parallel = 'None'
-        def num_machines = '1'
         def remoteTargets = [:]
         def additionalTestLabel = buildConfig.ADDITIONAL_TEST_LABEL
 
-        targets.each { target ->
-            // For each requested test, i.e 'sanity.jck', 'extended.jck', 'special.jck', call test job
+        // Determine from the platform the Jck jtx exclude platform
+        def excludePlat
+        def excludeRoot = "/home"
+        if (platform.contains("aix")) {
+            excludePlat = "aix"
+        } else if (platform.contains("mac")) {
+            excludePlat = "mac"
+            excludeRoot = "/Users"
+        } else if (platform.contains("windows")) {
+            excludePlat = "windows"
+            excludeRoot = "c:/Users"
+        } else if (platform.contains("solaris")) {
+            excludePlat = "solaris"
```

Comment:
is solairs using "/home" or "/export/home"  as `excludeRoot`

Reply:
good point, i'll check!

Reply:
yep changed, thanks